CommonData/columnutils.h

```diff
@@ -1,32 +1,53 @@
 #ifndef COLUMNUTILS_H
 #define COLUMNUTILS_H
 
+#include <functional>
```

Comment:
Hi @JorisGoosen , seems now you add it twice.

Reply:
Haha yeah I was afraid so, Im checking some last things :p